navigation-timing/resources/respond_slowly.py

```diff
@@ -0,0 +1,31 @@
+import time
+
+
+def main(request, response):
```

Comment:
I think the [trickle](https://web-platform-tests.org/tools/wptserve/docs/pipes.html?highlight=trickle#wptserve.pipes.trickle) pipeline can help you avoid a lot of this custom code ([usage example](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/web_tests/external/wpt/server-timing/navigation-timing-trickle.https.html?q=trickle%20f:wpt&ss=chromium))

Reply:
Huh. Might be worthwhile to get the WPT folks involved (and maybe file a bug). But agree we shouldn't wait on it